Kinvey/Kinvey/Client.swift

```diff
@@ -35,7 +35,6 @@ open class Client: Credential {
                 if let sharedKeychain = sharedKeychain {
                     try! sharedKeychain.removeAll()
                 }
-                dataStoreInstances.removeAll()
```

Comment:
Why was this removed?

Reply:
so this instance was holding instances as a cache, but this could cause problems, for example. You could create a `DataStore` instance with delta set disabled and then later on in the app you need a cache with delta set enabled, so that's why i think would be better to remove this cache object. Caching those objects does not make much of sense since the creating of new instances are really fast and cheap in terms of memory consumption.